test/layers/basic_tests.jl

```diff
@@ -9,6 +9,7 @@
             x = randn(rng, 6, 3) |> aType
 
             @test size(layer(x, ps, st)[1]) == (2, 3, 3)
+            @test Lux.outputsize(layer) == (2, 3)
```

Comment:
I'm not entirely sure if the definition of `outputsize` makes sense for `ReshapeLayer`.

Reply:
why?

Reply:
I'm a bit confused about the fact that the size of the output in the test above (and also in general) has a different size compared to `outputsize` because the input also matters and we don't know it from the layer definition.